src/RecipeApp.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import RecipeList from "./RecipeList"
+
+function RecipeApp() {
+    const initialValues = [{ "id": 1, "name": "Cake", "description": "deli", "ingredients": [{ "name": "milk" }] }];
```

Comment:
It's best to place constant values that never change outside of the react component. For example this initialValues